oap-storage-mongo/src/main/java/oap/storage/mongo/AbstractMongoTest.java

```diff
@@ -49,4 +65,28 @@ public void done() {
         mongoClient.database.drop();
         mongoClient.close();
     }
+
+    /**
+     * TODO: factor out it along with oap.storage.Bean from oap-storage module to {@link AbstractTest}
```

Comment:
Bean classes like these are only reusable within packages. They are look alike, true, but they bring locality to understanding what is important for the test. Universal testing beans eliminate this locality, and makes the test's ability to fix the behaviour worse.